kuma/settings/common.py

```diff
@@ -867,14 +815,19 @@ def JINJA_CONFIG():
 FLAG_REASONS = DEMO_FLAG_REASONS + WIKI_FLAG_REASONS
 
 # bit.ly
-BITLY_API_KEY = "SET ME IN SETTINGS_LOCAL"
-BITLY_USERNAME = "SET ME IN SETTINGS_LOCAL"
+BITLY_USERNAME = config('BITLY_USERNAME', default='lmorchard')
+BITLY_API_KEY = config('BITLY_API_KEY',
+                       default='R_2653e6351e31d02988b3da31dac6e2c0')
```

Comment:
Now that we have our own, should we update this too? (@hoosteeno - is there a way to get a dev/sandbox API key for our bitly account?) Or maybe we register for bitly from our mdn-dev@mozilla.com address and use that one?